.github/workflows/python-pure-wheels.yml

```diff
@@ -45,13 +45,12 @@ jobs:
 
 
       - name: Set up uv
-        if: ${{ steps.check-tag.outputs.run == 'true' }}
-        run: .github/script/install-uv.sh
-      - name: Set up Python
-        if: ${{ steps.check-tag.outputs.run == 'true' }}
-        uses: actions/setup-python@v5
+        uses: astral-sh/setup-uv@v3
         with:
-          python-version: '3.12'
+          version: "0.4.18"
```

Comment:
I feel an UV_VERSION envvar may be forgotten about, here we have it explicitly at the use point.
We don't really need to update this that often, the package manager should be stable enough.

I would prefer to just use the latest version all the time, but [they recommend pinning](https://github.com/astral-sh/setup-uv?tab=readme-ov-file#install-the-latest-version-default) to avoid unnecessary network traffic. 

I can do the env var if you prefer so

Reply:
set a different env var and use it explicitly here with something like `version: "{{env.TK2_UV_VERSION}}"` ?